frontend/app/views/spree/orders/edit.html.erb

```diff
@@ -16,7 +16,7 @@
         <div data-hook="inside_cart_form">
 
           <div data-hook="cart_items">
-            <%= render :partial => 'form', :locals => { :order_form => order_form } %>
+            <%= render 'form', order_form: order_form, order: @order %>
```

Comment:
Probably don't need to pass both `order_form` and `order` in, since `order` can be found via `order_form.object`.

If the goal is to make the code in the form partial nicer, it would be better to just store `order` as a local there.